### PR TITLE
✨ Added feature to NBA-game-viewer extension

### PR DESCRIPTION
**Pull request by AbhinavPalacharla**
…l information

## Description

Added new feature to NBA viewer extensions - can view team roster and player info

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/nba-game-viewer/src/components/Player.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+              <List.Item.Detail.Metadata.Label title="Salary" text={player.salary} />
+              <List.Item.Detail.Metadata.Separator />
+              <List.Item.Detail.Metadata.Label title="Draft Info" text={player.draft} />
+            </List.Item.Detail.Metadata>
+          }
+        />
+      }
+      actions={
+        <ActionPanel title="Player Actions">
+          <Action title="Show Player Info" icon={Icon.Sidebar} onAction={() => setIsShowingDetail(true)} />
```
Comment (pernielsentikaer) [inline extensions/nba-game-viewer/src/components/Player.tsx:44]: Maybe it should be a toggle instead?

File: extensions/nba-game-viewer/src/components/Player.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      title={player.fullName}
+      icon={player.headshot}
+      subtitle={player.position}
+      accessories={generatePlayerAccessories(player)}
+      detail={
+        <List.Item.Detail
+          markdown={`<img src="${player.headshot}" alt="drawing" width="250"/>`}
+          metadata={
+            <List.Item.Detail.Metadata>
+              <List.Item.Detail.Metadata.Label title="Position" text={player.position} />
```
Comment (pernielsentikaer) [inline extensions/nba-game-viewer/src/components/Player.tsx:23]: Maybe add the player name in the first row, I can't see the name
CleanShot 2022-06-04 at 07 52 05@2x

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @AbhinavPalacharla 👋 

What an excellent update, looking forward to using it myself 🔥 

I added some suggestions and feedback for you to address 💪 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by AbhinavPalacharla** [comment]
@pernielsentikaer I'm glad you liked the update and I'm excited for you to start using it. I added the changes you suggested and I have a couple more features planned.

**Review by pernielsentikaer — APPROVED** [review]
Hi @AbhinavPalacharla 👋 

Awesome update, thanks for taking your time 🔥 

I changed a few things, I hope you don't mind! Let's show player information from start 💪

**Pull request merged**